tests/unit/test_connection_overrides.py

```diff
@@ -140,7 +157,7 @@ def test_synapse_without_weight(create_tmp_simulation_config_file):
                     "name": "A2B",
                     "source": "RingA",
                     "target": "RingB",
-                    "modoverride": "AMPANMDA"
+                    "modoverride": "GABAAB",
```

Comment:
Does the test also pass with `"modoverride": "GluSynapse"` ? 
That's so far the only case that scientists use for 'modoverride', maybe good to test here.

Reply:
let me try

Reply:
not so simple. I get this error:
```
attribute = 'Use_d_TM', optional = False

    def _read(attribute, optional=False):
        if attribute in self._population.attribute_names:
            return self._population.get_attribute(attribute, needed_edge_ids)
        if optional:
            log_verbose("Defaulting to -1.0 for attribute %s", attribute)
            return -1
>       raise AttributeError(f"Missing attribute {attribute} in the SONATA edge file")
E       AttributeError: Missing attribute Use_d_TM in the SONATA edge file

/Users/katta/OBI/neurodamus/neurodamus/io/synapse_reader.py:325: AttributeError
```

If you want I can try to add that attribute to the edge file. What do you think?

Reply:
Ah, I see. Because additional fields are requested from the `GluSynapseHelper` https://github.com/openbraininstitute/neurodamus-models/blob/main/common/hoc/GluSynapseHelper.hoc#L12
And they are missing in the ringtest edges file.
Shall we just check an exception is raised in this case with `pytest.raises` ? Then we also cover the functionality of reading additional attributes from hoc file (although it fails).

Reply:
ah this is a good idea, yes